### ISSUE TITLE
classifier does not scan files recursively and organize them

### ISSUE THREAD

**Issue by rihbyne** [bhrigu123/classifier#68]
need to add code to scan files recursively with depth precision as an argument.

**Comment by premsai-ch** [comment]
I think this will be a great feature. Can I add this feature?

**Comment by JawadS** [comment]
Yep same here, voting for this feature, would be awesome if the app is run & monitors the folder for changes, or if not possible then running periodically to sort files..
